test/api.player.test.js

```diff
@@ -745,51 +745,51 @@ describe("Player API Tests", function() {
 
 		it("Test give badge", function(done) {
 			Playbasis.engineApi.rule("fbcomment", sentPlayerId, {post_custom_params: {type: "badge"}})
-				.then((result) => {
-					console.log(result);
-					expect(result.response.events.length == 1).toBeTruthy();
-					expect(result.response.events[0].reward_data.badge_id).not.toBe(null);
-					return api.giveGift(sentPlayerId, receivedPlayerId, result.response.events[0].reward_data.badge_id, "badge", 1);
-				}, (e) => { console.log(e.message); })
-				.then((result) => {
-					done();
-				}, (e) => { console.log(e.message); });
+			.then((result) => {
+				console.log(result);
+				expect(result.response.events.length == 1).toBeTruthy();
+				expect(result.response.events[0].reward_data.badge_id).not.toBe(null);
+				return api.giveGift(sentPlayerId, receivedPlayerId, result.response.events[0].reward_data.badge_id, "badge", 1);
+			}, (e) => { console.log(e.message); })
+			.then((result) => {
+				done();
```

Comment:
Just slightly more, please indented sub-sequence line inside `{}` to be indented from its parent properly.